### DISCUSSION TITLE
srsRAN 23.10.1v : oneplus8T disconnects after a few minutes.

### DISCUSSION THREAD

**Discussion by GershomSandeep** [srsran/srsRAN_Project#463] [General Help]
It connects correctly with the open5gs 2.7.0v core and srsRAN 23.10.0v that I am using (I followed the 
srsRAN gNB with COTS UEs document).
With srsRAN, most recent version 23.10.1, I'm having trouble (disconnects after a few minutes) with oneplus8T (unable to set an infinite timeout or disable the feature). Please advise me on how to configure the IMS for my Oneplus 8T.
FYI: I didn't use any external clocks for USRP B210.

**Comment by robertfalkenberg** [comment]
Maybe this can help: https://github.com/srsran/srsRAN_Project/discussions/135#discussioncomment-6521094

**Reply by pgawlowicz** [reply]
@GershomSandeep any updates on your issue?

**Comment by GershomSandeep** [comment]
No, I am still working on this. FYI, its working fine with srsRAN 23.10.0v.

On Tue, Feb 20, 2024 at 3:50 PM Piotr ***@***.***> wrote:

> @GershomSandeep <https://github.com/GershomSandeep> any updates on your
> issue?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/srsran/srsRAN_Project/discussions/463#discussioncomment-8527916>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADIBXVPY7C2FMXECGWWWSB3YUR2IHAVCNFSM6AAAAABDBDLTUSVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DKMRXHEYTM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Gershom Sandeep

**Reply by pgawlowicz** [reply]
do you run some traffic to/from the UE? if not, then the UE will be simply released due to inactivity.
